docs/installation.rst

```diff
@@ -117,6 +117,70 @@ the old version using the same steps as shown above.
 .. _pyinstaller: http://www.pyinstaller.org
 .. _releases: https://github.com/borgbackup/borg/releases
 
+.. _platforms:
+
+Supported platforms
+-------------------
+
+.. Yes/No's are grouped by reason/mechanism/reference.
+
++-------------------+----------+----------+-----------+------------+
+| Platform          | Tested   | ACLs     | xattr     | Flags      |
+|                   | [#tst]_  | [#acls]_ | [#xattr]_ | [#flags]_  |
++===================+==========+==========+===========+============+
+| Linux x86         | Yes [1]_ | Yes      | Yes       | Yes [6]_   |
++-------------------+----------+          |           |            |
+| Linux PowerPC     | Yes [2]_ |          |           |            |
++-------------------+----------+          |           |            |
+| Linux ARM         | Manually |          |           |            |
++-------------------+----------+----------+-----------+------------+
+| Mac OS X          | Yes [1]_ | Yes      | Yes       | Yes (all)  |
++-------------------+----------+----------+-----------+            |
+| FreeBSD           | Yes [2]_ | Yes      | Yes       |            |
++-------------------+----------+----------+-----------+            |
+| OpenBSD           | Yes [2]_ | n/a      | n/a       |            |
++-------------------+----------+----------+-----------+            |
+| NetBSD            | Manually | No [5]_  | No [7]_   |            |
++-------------------+----------+----------+           +------------+
+| Solaris 11        | Yes [2]_ | No       |           | n/a        |
++-------------------+----------+----------+           +------------+
+| OpenIndiana       | No       | No       |           | n/a        |
++-------------------+----------+----------+-----------+------------+
+| Windows (native)  | Yes [3]_ | Yes      | ?         | ?          |
++-------------------+----------+----------+-----------+------------+
+| Windows (cygwin)  | Manually | No [4]_  | No        | No         |
++-------------------+----------+----------+-----------+------------+
```

Comment:
some of these don't really fit under headline "supported platforms", rather "other platforms that might work".